core/src/css/component/battlegrounds/battles/bgs-simulator-hero-power-selection.component.scss

```diff
@@ -112,8 +112,10 @@
 				border: none;
 				cursor: text;
 				flex-grow: 1;
-				color: var(--color-3);
 
+				&::placeholder{
+					color: var(--color-2);
```

Comment:
Could you maybe add a lower opacity to make it stand out less and differentiate it more from the actual text?